config/Config-build.in

```diff
@@ -386,4 +386,15 @@ menu "Global build settings"
 
 	endchoice
 
+	config APPARMOR
+		bool "Enable AppArmor"
+		select KERNEL_SECURITY_APPARMOR
+		select KERNEL_AUDIT
+		select KERNEL_SECURITY
+		select KERNEL_SECURITYFS
+		select PACKAGE_apparmor
```

Comment:
This would need to be in openwrt.git to depend on it, so please remove this line. Whoever wants to use it has to manually install it.

Reply:
Ok.. It cannot be in openwrt.git since it requires python, which again is on packages. I'll remove that line..

Reply:
Also, this menu is for kernel configuration and userland software has nothing to do with kernel support, software on the other hand, does not work without kernel support, and then again, kernel support isn't that useful without software.. ;)
Committed.